<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateVisiteurBox</class>
 <widget class="QDialog" name="updateVisiteurBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>401</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>401</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color:#6b6b6b;
border:none;
border-radius:10px;</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Boite de modification&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Reset">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Reiniatialser les champs</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>411</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom du visiteur :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nomVisiteurEdit">
      <property name="styleSheet">
       <string notr="true">border-top:none;
                  border-left:none;
                  border-right:none;
                  border-bottom:1.5px solid #00a086;
                  margin-bottom:15px;
                  padding-bottom:5px;
                  min-height:24px;
border-radius:0;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="messageNomVisiteur">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Adresse :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="adresseVisiteurEdit">
      <property name="styleSheet">
       <string notr="true">border-top:none;
                  border-left:none;
                  border-right:none;
                  border-bottom:1.5px solid #00a086;
                  margin-bottom:15px;
                  padding-bottom:5px;
                  min-height:24px;
border-radius:0;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MsgAddresse">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Provenance :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="provenance">
      <property name="styleSheet">
       <string notr="true">border-top:none;
                  border-left:none;
                  border-right:none;
                  border-bottom:1.5px solid #00a086;
                  margin-bottom:15px;
                  padding-bottom:5px;
                  min-height:24px;
border-radius:0;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MsgProvenance">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmUpdateSiteBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#00A086;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
